EIPS/eip-5749.md

```diff
@@ -64,17 +65,17 @@ The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL
 
 ```
 
-Type `EIP1193Provider` is well documented at [EIP-1193](./eip-1193.md)
+Type `EIP1193Provider` is documented at [EIP-1193](./eip-1193.md)
 
-```
+```text
 interface ProviderInfo
 uuid: UUIDv4 unique to the provider
 name: Name of the Wallet
 icon: base64 encoded svg image
 description: Description for your wallet
 ```
```

Comment:
What you have is non-standard notation. I'd recommend using typedoc, which is standardized. In addition, the properties are underspecified:
```suggestion
```typescript
/**
 * Represents the assets needed to display a wallet
 */
interface ProviderInfo {
   /**
    * A UUIDv4 unique to the wallet provider.
    *
    * This must remain the same across versions but must be different across channels. For example, MetaMask, MetaMask Flask, and Trust Wallet should each have different UUIDs, but MetaMask 10.22.2 and MetaMask 9.8.1 should have the same UUID.
    *
    * @readonly
    */
    uuid: string;
    /**
     * The name of the wallet provider (e.g. `MetaMask` or `Trust Wallet`)
     *
     * @readonly
     */
    name: string;
    /**
     * A base64 encoded SVG image.
     *
     * Base64 is defined in RFC 4648.
     *
     * @readonly
     */
    icon: string;
    /**
     * A description of the wallet provider.
     *
     * @readonly
     */
    description: string;
}
```
```